packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -748,7 +753,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
     //vote. If you stake during an active reveal then your liquidity will be marked as inactive within Staker.sol until
     // the its activated in the next round and as such you'll miss out on being slashed for that round.
     function _getStartingIndexForStaker() internal override returns (uint64) {
-        _resolveResolvablePriceRequests();
+        _resolveResolvablePriceRequests(pendingPriceRequestsIds.length);
```

Comment:
This could have not the expected behavior if not all requests are traversed, see comment on `_resolveResolvablePriceRequests`